package.json

```diff
@@ -56,7 +56,7 @@
 		"yup": "^0.32.11"
 	},
 	"peerDependencies": {
-		"@lifesg/react-design-system": "^2.8.0-canary.4",
+		"@lifesg/react-design-system": "^2.9.0-canary.2",
```

Comment:
update the [documented version](https://github.com/LifeSG/web-frontend-engine/blob/main/README.md?plain=1#L15) in the readme file too